frameworks/elastic/tests/test_searchguard.py

```diff
@@ -0,0 +1,152 @@
+from typing import Any, Dict, Iterator
+
+import pytest
+import sdk_cmd
+import sdk_install
+import sdk_hosts
+import sdk_security
+
+from tests import config
+from security import transport_encryption
+
+sg_internal_users_secret_name = "sg_internal_users"
+
+
+def _uninstall_services() -> None:
+    sdk_install.uninstall(config.PACKAGE_NAME, config.SERVICE_NAME)
+
+
+@pytest.fixture
+def sg_internal_users_path(tmp_path) -> str:
+    path = tmp_path / "sg_internal_users.yml"
+    sg_internal_users_config = (
+        # password: admin_password
+        "admin.readonly: true\n"
+        'admin.hash: "$2y$12$Pd3kIQD1WgaKpekPyMkUi.jmBDF3QDmPIEUg37wXCRufZZQOnYYYW"\n'
+        "admin.roles: [ admin ]\n"
+        # password: kibanaserver_password
+        "kibanaserver.readonly: true\n"
+        'kibanaserver.hash: "$2y$12$lrwJYnjrlTGgOcf7Kd6xXOHIbkqFWBO3qgqyLYlEwIuRM3CfgL5fG"\n'
+    )
+
+    path.write_text(sg_internal_users_config)
+    return path.resolve()
+
+
+@pytest.fixture(scope="module")
+def service_account(configure_security: None) -> Iterator[Dict[str, Any]]:
```

Comment:
it would be nice to wrap fixture with this implementation 
https://github.com/mesosphere/dcos-elastic-service/blob/master/frameworks/elastic/tests/commons/tls.py#L10

Reply:
thanks! will do